packages/app/src/cli/services/app-logs/dev/write-app-logs.ts

```diff
@@ -12,10 +12,13 @@ interface AppLogFile {
 
 export const writeAppLogsToFile = async ({
   appLog,
+  appLogPayload,
   apiKey,
   stdout,
 }: {
   appLog: AppLogData
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  appLogPayload: AppLogPayload | any
```

Comment:
I'm not super happy with this type, but I feel it kind of conveys what we're expecting here?
The payload is `JSON.parse`'d and if the `appLog.log_type` is one we know, we convert it appropriately before passing it to the write. If we don't know it, we just pass the JSON `object`.